detail.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django_mako_plus import view_function, jscontext
+from datetime import datetime, timezone
+from catalog import models as cmod
+
+def fix_html(x):
```

Comment:
add comments to explain the purpose of this function

Reply:
Definitely.